build.gradle.kts

```diff
@@ -99,60 +99,15 @@ subprojects {
     jvmTarget = "1.8"
   }
 
-  sourceSets {
-    create("intTest") {
-      compileClasspath += sourceSets.main.get().output
-      runtimeClasspath += sourceSets.main.get().output
-    }
-  }
-
-  val intTestImplementation by configurations.getting {
-    extendsFrom(configurations.implementation.get())
-  }
-  val intTestRuntimeOnly by configurations.getting
-
-  configurations["intTestRuntimeOnly"].extendsFrom(configurations.runtimeOnly.get())
-
   dependencies {
     detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.23.4")
     detektPlugins("com.github.TBD54566975:tbd-detekt-rules:v0.0.2")
 
     testImplementation("org.junit.jupiter:junit-jupiter:5.9.2")
     testRuntimeOnly("org.junit.platform:junit-platform-launcher")
 
-    intTestImplementation(kotlin("test"))
-    intTestImplementation("org.junit.jupiter:junit-jupiter:5.9.2")
-    intTestRuntimeOnly("org.junit.platform:junit-platform-launcher")
   }
 
-  idea {
-    module {
-      testSources.from(sourceSets["intTest"].java.srcDirs)
-      testSources.from(sourceSets["intTest"].kotlin.srcDirs)
-    }
-  }
-
-  val integrationTest = task<Test>("integrationTest") {
-    description = "Runs integration tests."
-    group = "verification"
-
-    testClassesDirs = sourceSets["intTest"].output.classesDirs
-    classpath = sourceSets["intTest"].runtimeClasspath
-    shouldRunAfter("test")
-
-    useJUnitPlatform()
-
-    testLogging {
-      events("passed", "skipped", "failed", "standardOut", "standardError")
-      exceptionFormat = TestExceptionFormat.FULL
-      showExceptions = true
-      showCauses = true
-      showStackTraces = true
-    }
-  }
-
-  tasks.check { dependsOn(integrationTest) }
```

Comment:
Are the removal changes in this `build.gradle.kts` here as a byproduct of rolling our own DID types? 

(making sure the code changes reflect the intention of the PR)